doc/md/Use_of_Proxmark/4_Advanced-compilation-parameters.md

```diff
@@ -113,6 +113,7 @@ Here are the supported values you can assign to `STANDALONE` in `Makefile.platfo
 | LF_SKELETON     | standalone mode skeleton - Iceman1001
 | LF_THAREXDE     | LF EM4x50 simulator/read standalone mode - tharexde
 | HF_14ASNIFF     | 14a sniff storing to flashmem - Micolous
+| HF_15693SNIFF   | 15693 sniff storing to flashmem - Glaser
```

Comment:
Please rename it "HF_15SNIFF" as for 14a

Reply:
Done